src/SelectPanel/SelectPanel.tsx

```diff
@@ -153,18 +152,19 @@ export function SelectPanel({
       focusTrapSettings={focusTrapSettings}
       focusZoneSettings={focusZoneSettings}
     >
-      <Box display="flex" flexDirection="column" width="100%" height="100%">
-        <FilteredActionList
-          filterValue={filterValue}
-          onFilterChange={onFilterChange}
-          {...listProps}
-          role="listbox"
-          items={itemsToRender}
-          selectionVariant={isMultiSelectVariant(selected) ? 'multiple' : 'single'}
-          textInputProps={extendedTextInputProps}
-          inputRef={inputRef}
-        />
-      </Box>
+      <FilteredActionList
+        filterValue={filterValue}
+        onFilterChange={onFilterChange}
+        {...listProps}
```

Comment:
Since the `FilteredActionListProps` interface now extends `SxProp`, someone may pass a `listProps` that includes `sx`, right? If they do, line 166 will overwrite the prop `sx` value they pass, right? Would we want to merge?

Reply:
@jfuchs I might be missing something, but it seems that `SelectPanelProps` _does_ contain `sx`, via `Omit<FilteredActionListProps, 'selectionVariant'>`: https://github.com/primer/react/blob/c4926a26034b42fced5de4583169d659d010cac4/src/SelectPanel/SelectPanel.tsx#L34-L35

<img width="479" alt="Screen Shot 2021-10-06 at 3 40 49 PM" src="https://user-images.githubusercontent.com/3104489/136272224-b412035f-26ba-4b64-bc6e-9df39feb5b0c.png">